__init__.py

```diff
@@ -6,4 +6,4 @@
 """__init__ for mstic.py package."""
 
 # flake8: noqa: F403
-from . msticpy._version import VERSION as __version__
```

Comment:
After I merge TIProvider PR you should probably re-merge with master and that should make all of black formatting diffs disappear.

Reply:
Done, remerged from master.